cmd/status.go

```diff
@@ -0,0 +1,308 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/fatih/color"
+	"github.com/mattn/go-isatty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+var (
+	// taken from https://github.com/fatih/color/blob/4d2835ff85a014514ee435d49f76dc8b25c9cee3/color.go#L20-L21
+	noColor = os.Getenv("TERM") == "dumb" ||
+		(!isatty.IsTerminal(os.Stdout.Fd()) && !isatty.IsCygwinTerminal(os.Stdout.Fd()))
+)
+
+const (
+	productionURL = "https://epr.elastic.co"
+	stagingURL    = "https://epr-staging.elastic.co"
+	snapshotURL   = "https://epr-snapshot.elastic.co"
+)
+const statusLongDescription = `Use this command to display the current deployment status of a package.
+
+If a package name is specified, then information about that package is
+returned, otherwise this command checks if the current directory is a
+package directory and reports its status.`
+
+func setupStatusCommand() *cobraext.Command {
+	cmd := &cobra.Command{
+		Use:   "status [package]",
```

Comment:
nit: `"status"`

You can give a sample use case with a package in the long description.

Reply:
This, as far as I know, is the only way to have cobra properly set the Usage line for a subcommand (i.e. `elastic-package status [package] [flags]`) with additional, non-flag arguments -- removing this will cause the `Usage` line to change. Is that alright?

Reply:
Also, see the example in the official README: https://github.com/spf13/cobra#example where it has

```go
    Use:   "print [string to print]",
```

Reply:
Thanks for bringing it up. It's a fair point. Please ignore the comment and apologize for the concern.